tests/unittests/thermal_heuristic/test_model.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+from andromede.expression import literal, param, var
+from andromede.model import model
+from andromede.model.constraint import Constraint
+from andromede.thermal_heuristic.model import ModelEditor
+
+
+def test_variable_in_expression() -> None:
+    model_editer = ModelEditor(model("model"))
+
+    expression_1 = var("x").sum() + var("y")
+    expression_2 = (
+        var("z").shift(-1) + var("z") + literal(0) * var("y") + var("x").expec()
+    )
+
+    assert model_editer.variable_in_expression(expression_1, ["x"]) is True
```

Comment:
`is True` is not needed, assert will already check that the given expression is True